### DISCUSSION TITLE
Quarkus Theming importing custom common

### DISCUSSION THREAD

**Discussion by snovak7** [keycloak/keycloak#10557] [Q&A]
Hello,

I'm trying to have some kind of common resources, and I looked up to the keycloak theme, where they

`import=common/keycloak`

So I've tried to do the same with my theme, currently I'm editing the welcome type

`import=common/<theme>`

But the output is still as with the keycloak

Is this possible to do?

I'm currently developing with themes folder directly on the server, I also tried to do the theme package, but then I just get the error from keycloak. Exceptions/Errors are not really informative if properties are not properly written.

**Comment by snovak7** [comment]
I see that it is hardcoded: https://github.com/keycloak/keycloak/blob/main/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/services/resources/QuarkusWelcomeResource.java#L183

**Reply by pculka** [reply]
stumbled upon this aswell... What's the point of import in theme.properties, then?
